$NetBSD$

Defines PTHREAD_MUTEX_RECURSIVE_NP and PTHREAD_MUTEX_RECURSIVE if not already.

--- src/threads/mutex.cpp.orig	2007-01-07 07:47:34.000000000 +0000
+++ src/threads/mutex.cpp
@@ -23,6 +23,10 @@
 
 using namespace std;
 
+#ifndef PTHREAD_MUTEX_RECURSIVE_NP
+#define PTHREAD_MUTEX_RECURSIVE_NP PTHREAD_MUTEX_RECURSIVE
+#endif
+
 ///////////////////////////
 // t_mutex
 ///////////////////////////
